docs/core/prerequisites.md

```diff
@@ -0,0 +1,453 @@
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+# [.NET Core 3.0](#tab/netcore30)
+
+.NET Core 3.0 is supported on **macOS High Sierra (version 10.13)** and later versions. An **x64** CPU architecture is required.
+
+For more information about .NET Core 3.0 supported OS, distributions, and lifecycle policy, see [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md).
+
+# [.NET Core 2.2](#tab/netcore22)
+
+.NET Core 2.2 is supported on **macOS Sierra (version 10.12)** and later versions. An **x64** CPU architecture is required.
+
+For more information about .NET Core 2.2 supported OS, distributions, and lifecycle policy, see [.NET Core 2.2 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.2/2.2-supported-os.md).
+
+# [.NET Core 2.1](#tab/netcore21)
+
+.NET Core 2.1 is supported on **macOS Sierra (version 10.12)** and later versions. An **x64** CPU architecture is required.
+
+For more information about .NET Core 2.1 supported OS, distributions, and lifecycle policy, see [.NET Core 2.1 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md).
+
+---
+
+::: zone-end
+
+::: zone pivot="os-windows"
+
+## Install with Visual Studio
+
+Visual Studio installs the latest .NET Core SDK tools, and one or more runtimes so you can target your app to run on the .NET Core runtime version of your choice.
+
+Selecting any of the managed workloads for **Web**, **Desktop**, or **Azure development**, will install the .NET Core development tools and runtime. Specifically choosing the **.NET Core** workload also installs .NET Core.
+
+[![Windows Visual Studio 2019 with .NET Core workload](media/prerequisites/windows-install-visual-studio-2019.png)](media/prerequisites/windows-install-visual-studio-2019.png#lightbox)
+
+After you've installed Visual Studio, create your first app by following the [C# Hello World tutorial](tutorials/with-visual-studio.md) or the [Visual Basic Hello World tutorial](tutorials/vb-with-visual-studio.md).
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+## Install with Visual Studio for Mac
+
+Visual Studio for Mac installs the .NET Core SDK when the **.NET Core workload** is selected. To get started with .NET Core development on macOS, see [Install Visual Studio 2019 for Mac](https://docs.microsoft.com/visualstudio/mac/installation?view=vsmac-2019).
+
+[![macOS Visual Studio 2019 for Mac with .NET Core workload feature](media/prerequisites/mac-install-selection.png)](media/prerequisites/mac-install-selection.png#lightbox)
+
+After you've installed Visual Studio for Mac, create your first app by following the [Get started on macOS](tutorials/using-on-mac-vs.md) tutorial.
+
+::: zone-end
+
+## Install with Visual Studio Code
+
+Visual Studio Code (VS Code) is a powerful and lightweight source code editor that runs on your desktop. VS Code is available for Windows, macOS, and Linux.
```

Comment:
In theory, we should never use VS Code (https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=40682). I'd hesitate to clearly shorten to VS Code like this. The way that the VS Code docs folks do is that they simply use VS Code after the full name was mentioned.

```suggestion
Visual Studio Code is a powerful and lightweight source code editor that runs on your desktop. VS Code is available for Windows, macOS, and Linux.
```